### ISSUE TITLE
Quick-add "React" should include "ReactDOM"

### ISSUE THREAD

**Issue by HipsterBrown** [CodePen/CodePen-Bug-Reporting-Only#250]
As of React v0.14, using React on the web requires two libraries: `react` && `react-dom`. The new `react-dom` library is used for interacting with the DOM, so that the core `react` library focuses on the Component API shared by any platform, i.e. React Native. 

When using the "Quick-add" feature in "Pen Settings: JavaScript" and selecting "React", it only includes the core `react` library, so the usual `React.render(<Component />, DOMNode)` does not work. For this to work, selecting "React" also adds a link to the `cdnjs` copy of `react-dom` (https://cdnjs.cloudflare.com/ajax/libs/react/15.0.2/react-dom.min.js) in the 2nd "External JavaScript" field available. OR have another option available in the list for "ReactDOM". 

TL;DR React on the web needs the `react` and `react-dom` libraries to work. Both should be included when selecting "React" in the "Quick-add" dropdown. 👍 

Cheers and thanks for making CodePen awesome every day!

**Comment by chriscoyier** [comment]
Already have this done and waiting to go out in our next deploy.

![bitmoji](https://render.bitstrips.com/v2/cpanel/7136645-122222069_10-s1-v1.png?palette=1&width=246)

**Comment by HipsterBrown** [comment]
You da best! :clap: 🎉
